<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>67</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>MGRS panel</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MGRS Coordinate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leMgrsCoordinate"/>
    </item>
    <item>
     <widget class="QToolButton" name="btnZoom">
      <property name="toolTip">
       <string>Zoom to MGRS coordinate</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btnRemoveMarker">
      <property name="toolTip">
       <string>Remove marker from canvas</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
